README.md

```diff
@@ -47,7 +47,7 @@ npm link
 Starts local testnet with 4 relay chain validators and Basilisk as parachain
 
 ```
-cd rococo-local
+cd ../rococo-local
```

Comment:
rococo local is located in repo as subdirectory
```suggestion
cd rococo-local
```

Reply:
Sure. The issue comes from the section above where we move to the polkadot-local dir where the rococo-local dir does not exist.